ubisense_uwb_publisher:
  ros__parameters:
    udp_ip: "127.0.0.1" # UDP_IP = "192.168.10.24" # My laptop's IP when I connect to CranfieldIOT
    udp_port: 38633
    tag: "0011ce000000f923"
    debug: true # 0 connect to Ubisense, 1 not connect to Ubisense and use a dummy message to spin the engine 